docs/modules/ROOT/pages/explanation/decisions/kyverno.adoc

```diff
@@ -0,0 +1,25 @@
+= Kyverno as policy engine
+:apub2: APPUiO Public 2.0
+
+Problem::
+Kubernetes RBAC rules are not suitable to validate certain properties of resources.
+They can only allow or deny operations of the whole resource.
+On {apub2} however, we require fine-grained control of certain properties, such as labels or annotations on Namespaces.
+
+//Relevant requirements::
+//* tbd, links to requirement page(s)
+
+Alternatives::
+* https://www.openpolicyagent.org/[OPA].
+  Open Policy Agent is a policy engine written for cloud native environments.
+  Policies are written in Rego, which is declarative query language suited for writing policies.
+
+* https://kyverno.io/[Kyverno].
+  Kyverno is a Kubernetes native policy engine that uses Kyverno-specific CRDs to configure policies.
+
+Decision::
+Kyverno
+
+Rationale::
```

Comment:
Maybe also add a point that Kyverno is able to generate resources which OPA can't.

Reply:
Not sure... IMHO we should mention this, even if it solved a different problem and mention it accordingly.

Reply:
It's irrelevant for the policy engine decision. If Kyverno would not be able to generate resources, we'd still choose Kyverno and generate resources with other options.

But I have added the decision for generating resources and linked them together. Is that acceptable?